packages/client/src/arpa_reporter/router/index.js

```diff
@@ -1,8 +1,8 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
 
-import Agencies from '../views/Agencies.vue';
-import Agency from '../views/Agency.vue';
+import AgenciesView from '@/arpa_reporter/views/AgenciesView.vue';
+import AgencyView from '@/arpa_reporter/views/AgencyView.vue';
 import Home from '../views/Home.vue';
 import Login from '../views/Login.vue';
 import NewTemplate from '../views/NewTemplate.vue';
```

Comment:
Same question: why are only some of these import paths updated?  Is this just an artefact of breaking apart these changes into separate PRs?

Reply:
See my answer here: https://github.com/usdigitalresponse/usdr-gost/pull/2847#discussion_r1541410247